guides/tutorials/mediation.md

```diff
@@ -0,0 +1,120 @@
+# Mediation
+
+> This section assumes that you have:
+>
+> 1. Set up a [valid environment](../getting-started/installation) for development
+>
+> 1. Basic knowledge of the required fields in the [agent
+>    config](./agent-config)
```

Comment:
```suggestion
> 1. Set up a [valid environment](../getting-started/installation) for development
> 1. Basic knowledge of the required fields in the [agent
>    config](./agent-config)
```

Reply:
We should probably update this to be a bit more sexy, maybe using an admonition. Not for this PR as it's also present in the other tutorials